### DISCUSSION TITLE
Unable to install PIMCORE Enterprises on the local Windows machine.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15761] [Q&A / Support]
Pimcore Enterprises is up and running on our Ubuntu server. 

We try to achieve code management :

1. Pimcore is configured locally for development changes and testing. it's never interferes with the Sandbox or UAT server.
2. Code repository management for pull, push, commit, review and code merge approvals.
3. The latest code modifications released from the repository to Sandbox or UAT server.

Current sandbox details:
Pimcore Version : 10.5.21
PHP version : 8.0.28
Elasticsearch 7.10.2

Windows 11 Configuration details:
Xampp -Installed
PHP version : 8.2.4 -Default by Xampp
Elasticsearch 7.10.2 -Installed with java
I created a backup from the PIM sandbox on the pimcore database, and I restored it to a new Xampp database using the pimcore name.
The xampp directory was set up by the PIM sandbox code base.
![image](https://github.com/pimcore/pimcore/assets/106297450/ee944939-44f7-4e09-8e8d-c84ab2ec832a)
![image](https://github.com/pimcore/pimcore/assets/106297450/5526f325-6b20-4c23-9662-2f93bc530d94)

I started my Xampp server and mysql :
![image](https://github.com/pimcore/pimcore/assets/106297450/1119693a-37be-4bb0-8e3c-7c403fff6f39)

created a new user with the admin role on Elasticsearch :
![image](https://github.com/pimcore/pimcore/assets/106297450/b02e2aae-d960-4fab-a27d-47cb386c1c1b)
![image](https://github.com/pimcore/pimcore/assets/106297450/f2d2973d-1f8f-49c2-a9e8-ac9f2cc0da8a)

started the Elasticsearch and status for your ref :
![image](https://github.com/pimcore/pimcore/assets/106297450/8b8718d1-80c5-4ac2-9d82-2d53f611cefa)
![image](https://github.com/pimcore/pimcore/assets/106297450/39776682-e4e0-4a4d-a8d2-e261200d235c)
![image](https://github.com/pimcore/pimcore/assets/106297450/da1f6f0f-ee42-4a2b-a24a-06d81acf8ba2)

The below configuration only enabled in the elastisearch.yaml file
![image](https://github.com/pimcore/pimcore/assets/106297450/588737f1-df54-4131-8bf7-3b3964a4aca2)

I am able to access the localhost Pimcore admin page :
![image](https://github.com/pimcore/pimcore/assets/106297450/e7d1bed4-6648-4234-b8f0-aefe4be58640)

unable to modify or save the outdated data objects!
(Status: 500 | Internal Server Error
URL: /admin/object/save?task=publish
Method: PUT
Message: {"error":{"root_cause":[{"type":"security_exception","reason":"missing authentication credentials for REST request [/datahub_restindex__product_products/_search?track_total_hits=true&rest_total_hits_as_int=true]","header":{"WWW-Authenticate":"Basic realm=\"security\" charset=\"UTF-8\""}}],"type":"security_exception","reason":"missing authentication credentials for REST request [/datahub_restindex__product_products/_search?track_total_hits=true&rest_total_hits_as_int=true]","header":{"WWW-Authenticate":"Basic realm=\"security\" charset=\"UTF-8\""}},"status":401}
Trace: 
in C:\xamppt\htdocs\pimcore\vendor\elasticsearch\elasticsearch\src\Elasticsearch\Connections\Connection.php:684)

Elasticsearch configuration on local pimcore setup :
![image](https://github.com/pimcore/pimcore/assets/106297450/694f577f-5805-4fa8-b5f5-321584507e2c)

I am unable to login to the Portal engine page if I login, it's only redirect same page http://localhost/auth/login.
![image](https://github.com/pimcore/pimcore/assets/106297450/ab8ea843-903c-4f31-8a43-b4fbb18cc0e2)

Anyone, Let me know if the configuration details are right or wrong.

How can I fix this issue?

@fashxp @kingjia90

**Comment by kingjia90** [comment]
Could be an issue with the `.env` file in the root  and defaulting `APP_ENV` to `prod`? IIRC Xampp manages the global ENV differently but it's been a long time that i don't use Xampp, got used to WSL2+Docker combination 😅 Windows also doesn't like files just starting with `.` or so

**Reply by arulselvan008** [reply]
Thanks ji.
